modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -782,7 +782,7 @@ class Editor implements EditorObservable {
    * @method setContent
    * @param {String} content Content to set to editor, normally HTML contents but can be other formats as well.
    * @param {Object} args Optional content object, this gets passed around through the whole set process.
-   * @return {String} HTML string that got set into the editor.
+   * @deprecated @return {String} HTML string that got set into the editor.
```

Comment:
This is going to break the docs, so we either need to mention this in the description or only mention the it's been deprecated via the changelog. The description is probably going to be the better answer, so maybe something like this?

```
  /**
   * Sets the specified content to the editor instance, this will cleanup the content before it gets set using
   * the different cleanup rules options.
   * <br>
   * <em>The content return value was deprecated in TinyMCE 6.0 and has been marked for removal in TinyMCE 7.0.</em>
   * 
   * @method setContent
```

- [ ] Fix this per the above